ads/js/sticky.js

```diff
@@ -15,14 +16,25 @@ Sticky.prototype.stick = function () {
 	if (this.sibling) {this.sibling.style.marginTop = this.el.offsetHeight + 'px';}
 };
 
+Sticky.prototype.unstick = function () {
+	this.el.style.position = 'absolute';
+	this.el.style.top = this.stickyUntilPoint + 'px'; //HEADER
+	// this.el.style.top = (this.stickyUntilPoint - this.el.offsetHeight) + 'px'; //RHR
+	this.sibling.style.marginTop = this.el.offsetHeight + 'px';
+};
+
 Sticky.prototype.onScroll = function () {
-	if((this.stickyUntilPoint > window.pageYOffset) && (window.pageYOffset > this.opts.stickAfter)) {
+	if(this.extraHeight === false && document.querySelector('.visible')) {
```

Comment:
Maybe make this more specific...I suspect there's a few things on the page with a class of .visible